packages/plugins/documentation/admin/src/pages/PluginPage/index.js

```diff
@@ -0,0 +1,61 @@
+/**
+ *
+ * This component is the skeleton around the actual pages, and should only
+ * contain code that should be seen on all pages. (e.g. navigation bar)
+ *
+ */
+
+import React from 'react';
+import { useIntl } from 'react-intl';
+import { CheckPermissions, LoadingIndicatorPage } from '@strapi/helper-plugin';
+import { Layout, HeaderLayout, Main, Button } from '@strapi/parts';
+import ShowIcon from '@strapi/icons/Show';
+
+import permissions from '../../permissions';
+import { getTrad } from '../../utils';
+import openWithNewTab from '../../utils/openWithNewTab';
+import useHomePage from '../HomePage/useHomePage';
+
+const PluginPage = () => {
+  const { formatMessage } = useIntl();
+  const { data, isLoading } = useHomePage();
+
+  if (isLoading) {
+    return <LoadingIndicatorPage />;
```

Comment:
Maybe you can add some context about what is being loaded here in order to give the information t the ATs.

Something like 🤷🏽 
```js
<LoadingIndicatorPage>Plugin is loading</LoadingIndicatorPage>
```